package.json

```diff
@@ -28,7 +28,7 @@
     "koa-route": "^2.4.0",
     "koa-statsd": "^0.0.2",
     "moment": "^2.8.4",
-    "mongoose": "3.8.23",
+    "mongoose": "^4.0.1",
```

Comment:
Should we keep the caret given the latest issues we've been getting? Or do you think it'll be okay with mongoose switching to semver? Maybe just a `~`?